crd/multitenancy/api/v1alpha1/multitenantpodnetworkconfig.go

```diff
@@ -51,16 +47,34 @@ type MultitenantPodNetworkConfigSpec struct {
 	PodName string `json:"podName,omitempty"`
 }
 
+type InterfaceInfo struct {
+	// NCID is the network container id
+	NCID string `json:"ncID,omitempty"`
+	// PrimaryIP is the ip allocated to the network container
+	PrimaryIP string `json:"primaryIP,omitempty"`
+	// MacAddress is the MAC Address of the VM's NIC which this network container was created for
+	MacAddress string `json:"macAddress,omitempty"`
+	// GatewayIP is the gateway ip of the injected subnet
+	GatewayIP string `json:"gatewayIP,omitempty"`
```

Comment:
we planned on adding NICType as a field too withing InterfaceInfos so singularity can directly start using & writing to `InterfaceInfos[`] instead of adding NICType in `MultiTenantPodNetworkConfigStatus`

Reply:
done